packages/design-system/src/components/Alert/Alert.test.tsx

```diff
@@ -41,8 +53,8 @@ describe('Alert', function () {
   it('renders additional className and role prop', () => {
     const className = 'ds-u-test';
     const role = 'alert';
-    renderAlert({ className, role });
-    const alert = screen.getByRole(role);
+    renderAlert({ 'class-name': className, role });
```

Comment:
web component props/attrs can only have lowercase and hyphenated names. our camelCase props don't work.